### ISSUE TITLE
Would the use of SSDT-AWAC-ARTC.asl affect the Windows system.

### ISSUE THREAD

**Issue by sunbos** [5T33Z0/OC-Little-Translated#66]
If I use SSDT-AWAC-ARTC.asl, I need to delete the HPET table. Would the double system affect the Windows system.
Would the use of SSDT-AWAC-ARTC.asl affect the Windows system？

**Comment by 5T33Z0** [comment]
This is already answered in the article about SSDT-ARTC…

In Windows 10 and newer versions of the operating system, **HPET is a legacy device and is not used**. Instead, Windows 10 uses a combination of the TSC (Time Stamp Counter) and the ACPI (Advanced Configuration and Power Interface) PM timer to keep track of time.

Some older applications or drivers may still rely on HPET, and in those cases you can enable HPET in the BIOS to make it available to the operating system. If you are not experiencing any issues with your system and you don't need to use any older applications that require HPET, you can generally leave it disabled to save power.

In other words: perform some tests on your to figure out it it affects your system.

**Comment by sunbos** [comment]
Thank you very much for your answer. It's very useful to me
